Makefile

```diff
@@ -1,8 +1,13 @@
 .PHONY: all tools test validate lint .gitvalidation fmt
 
 # Which github repository and branch to use for testing with skopeo
+ifdef TRAVIS_PULL_REQUEST_SLUG
+SKOPEO_REPO = $(subst image,skopeo,$(TRAVIS_PULL_REQUEST_SLUG))
+SKOPEO_BRANCH = $(TRAVIS_BRANCH)
+else
```

Comment:
This part modifies variables that are removed by another part in this PR; this seems to be a leftover from an abandoned approach.

Reply:
Not really, it was a bug that should get fixed with commit https://github.com/Silvanoc/image/commit/375b9cc8eccc088537932aa4c608d28d5b6cf6dd.